From b57a7bf0589e067a46ae31ceb825682512d74f6e Mon Sep 17 00:00:00 2001
From: Xiaoqiang Liu <leavs_o@126.com>
Date: Sat, 24 Aug 2019 10:35:52 +0800
Subject: [PATCH] update for clearly code

1. Remove panel args
2. change default mmcdev to 2, mmcroot to mmcblk3p2.
---
 include/configs/mx6eisd.h        | 4 ++--
 include/configs/mx6eisd_common.h | 1 -
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/include/configs/mx6eisd.h b/include/configs/mx6eisd.h
index 5b6eb8a..d62b0d3 100644
--- a/include/configs/mx6eisd.h
+++ b/include/configs/mx6eisd.h
@@ -23,7 +23,7 @@
 #define CONFIG_MACH_TYPE	3980
 #define CONFIG_MXC_UART_BASE	UART1_BASE
 #define CONFIG_CONSOLE_DEV		"ttymxc0"
-#define CONFIG_MMCROOT			"/dev/mmcblk2p2"  /* SDHC3 */
+#define CONFIG_MMCROOT			"/dev/mmcblk3p2"  /* SDHC4 */
 #if defined(CONFIG_MX6QP)
 #define CONFIG_DEFAULT_FDT_FILE	"imx6qp-eisd.dtb"
 #define PHYS_SDRAM_SIZE		(2u * 1024 * 1024 * 1024)
@@ -51,7 +51,7 @@
 #include "mx6eisd_common.h"
 
 #define CONFIG_SYS_FSL_USDHC_NUM	3
-#define CONFIG_SYS_MMC_ENV_DEV		1	/* SDHC3 */
+#define CONFIG_SYS_MMC_ENV_DEV		2	/* SDHC4 */
 #define CONFIG_SYS_MMC_ENV_PART                0       /* user partition */
 
 #ifdef CONFIG_SYS_USE_SPINOR
diff --git a/include/configs/mx6eisd_common.h b/include/configs/mx6eisd_common.h
index 835d223..34046f9 100644
--- a/include/configs/mx6eisd_common.h
+++ b/include/configs/mx6eisd_common.h
@@ -204,7 +204,6 @@
 	"console=" CONFIG_CONSOLE_DEV "\0" \
 	"fdt_high=0xffffffff\0"	  \
 	"initrd_high=0xffffffff\0" \
-	"panel=BOE185\0" \
 	"splashpos=m,m\0" \
 	"mmcdev=" __stringify(CONFIG_SYS_MMC_ENV_DEV) "\0" \
 	"mmcpart=" __stringify(CONFIG_SYS_MMC_IMG_LOAD_PART) "\0" \
-- 
2.7.4

